### PR TITLE
Rename method arguments to use camelCase casing (libraries/joomla directory)

### PR DESCRIPTION
**Pull request by SharkyKZ**
Partial Pull Request for Issue https://github.com/joomla/joomla-cms/issues/30988.

### Summary of Changes

Renames some method arguments to use camelCase casing.

### Testing Instructions

Thorough code review / Joomla works like before.

### Documentation Changes Required

No.

### CODE REVIEW

File: libraries/joomla/linkedin/people.php [hunk 1]
```diff
@@ -156,38 +156,39 @@ public function getConnections($fields = null, $start = 0, $count = 500, $modifi
...
+	 *                                    Not supported for all countries.
+	 * @param   integer  $distance        Matches members within a distance from a central point. This is measured in miles.
+	 * @param   string   $facets          Facet buckets to return, e.g. location.
+	 * @param   array    $facet           Array of facet values to search over. Contains values for location, industry, network, language,
+	 *                                    current-company, past-company and school, in exactly this order,
+	 *                                    null must be specified for an element if no value.
+	 * @param   integer  $start           Starting location within the result set for paginated returns.
+	 * @param   integer  $count           The number of results returned.
+	 * @param   string   $sort            Controls the search result order. There are four options: connections, recommenders,
+	 * 									  distance and relevance.
```
Comment (Quy) [inline libraries/joomla/linkedin/people.php:183]: Alignment.

File: libraries/joomla/linkedin/people.php [hunk 2]
```diff
@@ -156,38 +156,39 @@ public function getConnections($fields = null, $start = 0, $count = 500, $modifi
...
-	 * @param   integer  $distance         Matches members within a distance from a central point. This is measured in miles.
-	 * @param   string   $facets           Facet buckets to return, e.g. location.
-	 * @param   array    $facet            Array of facet values to search over. Contains values for location, industry, network, language,
-	 * 									   current-company, past-company and school, in exactly this order, null must be specified for an element if no value.
-	 * @param   integer  $start            Starting location within the result set for paginated returns.
-	 * @param   integer  $count            The number of results returned.
-	 * @param   string   $sort             Controls the search result order. There are four options: connections, recommenders,
-	 * 									   distance and relevance.
+	 * @param   string   $fields          Request fields beyond the default ones. provide 'api-standard-profile-request'
+	 * 	                                  field for out of network profiles.
```
Comment (Quy) [inline libraries/joomla/linkedin/people.php:160]: Alignment.

File: libraries/joomla/twitter/friends.php [hunk 3]
```diff
@@ -20,19 +20,19 @@ class JTwitterFriends extends JTwitterObject
...
 	 *
-	 * @param   mixed    $user        Either an integer containing the user ID or a string containing the screen name.
-	 * @param   integer  $cursor      Causes the list of connections to be broken into pages of no more than 5000 IDs at a time.
-	 * 								  The number of IDs returned is not guaranteed to be 5000 as suspended users are filtered out
-	 * 								  after connections are queried. If no cursor is provided, a value of -1 will be assumed, which is the first "page."
-	 * @param   boolean  $string_ids  Set to true to return IDs as strings, false to return as integers.
-	 * @param   integer  $count       Specifies the number of IDs attempt retrieval of, up to a maximum of 5,000 per distinct request.
+	 * @param   mixed    $user       Either an integer containing the user ID or a string containing the screen name.
+	 * @param   integer  $cursor     Causes the list of connections to be broken into pages of no more than 5000 IDs at a time.
+	 * 							     The number of IDs returned is not guaranteed to be 5000 as suspended users are filtered out
```
Comment (Quy) [inline libraries/joomla/twitter/friends.php:25]: Alignment.

File: libraries/joomla/twitter/friends.php [hunk 4]
```diff
@@ -133,19 +133,19 @@ public function getFriendshipDetails($user_a, $user_b)
...
-	 * @param   mixed    $user        Either an integer containing the user ID or a string containing the screen name.
-	 * @param   integer  $cursor      Causes the list of IDs to be broken into pages of no more than 5000 IDs at a time. The number of IDs returned
-	 * 								  is not guaranteed to be 5000 as suspended users are filtered out after connections are queried. If no cursor
-	 * 								  is provided, a value of -1 will be assumed, which is the first "page."
-	 * @param   boolean  $string_ids  Set to true to return IDs as strings, false to return as integers.
-	 * @param   integer  $count       Specifies the number of IDs attempt retrieval of, up to a maximum of 5,000 per distinct request.
+	 * @param   mixed    $user       Either an integer containing the user ID or a string containing the screen name.
+	 * @param   integer  $cursor     Causes the list of IDs to be broken into pages of no more than 5000 IDs at a time. The number of IDs returned
+	 *                               is not guaranteed to be 5000 as suspended users are filtered out after connections are queried. If no cursor
+	 * 								 is provided, a value of -1 will be assumed, which is the first "page."
```
Comment (Quy) [inline libraries/joomla/twitter/friends.php:139]: Alignment.

File: libraries/joomla/twitter/friends.php [hunk 5]
```diff
@@ -193,16 +193,16 @@ public function getFollowerIds($user, $cursor = null, $string_ids = null, $count
 	/**
 	 * Method to determine pending requests to follow the authenticating user.
 	 *
-	 * @param   integer  $cursor      Causes the list of IDs to be broken into pages of no more than 5000 IDs at a time. The number of IDs returned
-	 * 								  is not guaranteed to be 5000 as suspended users are filtered out after connections are queried. If no cursor
-	 * 								  is provided, a value of -1 will be assumed, which is the first "page."
-	 * @param   boolean  $string_ids  Set to true to return IDs as strings, false to return as integers.
+	 * @param   integer  $cursor     Causes the list of IDs to be broken into pages of no more than 5000 IDs at a time. The number of IDs returned
+	 * 								 is not guaranteed to be 5000 as suspended users are filtered out after connections are queried. If no cursor
```
Comment (Quy) [inline libraries/joomla/twitter/friends.php:197]: Alignment.

File: libraries/joomla/twitter/friends.php [hunk 6]
```diff
@@ -231,16 +231,16 @@ public function getFriendshipsIncoming($cursor = null, $string_ids = null)
 	/**
 	 * Method to determine every protected user for whom the authenticating user has a pending follow request.
 	 *
-	 * @param   integer  $cursor      Causes the list of IDs to be broken into pages of no more than 5000 IDs at a time. The number of IDs returned
-	 * 								  is not guaranteed to be 5000 as suspended users are filtered out after connections are queried. If no cursor
-	 * 								  is provided, a value of -1 will be assumed, which is the first "page."
-	 * @param   boolean  $string_ids  Set to true to return IDs as strings, false to return as integers.
+	 * @param   integer  $cursor     Causes the list of IDs to be broken into pages of no more than 5000 IDs at a time. The number of IDs returned
+	 * 								 is not guaranteed to be 5000 as suspended users are filtered out after connections are queried. If no cursor
```
Comment (Quy) [inline libraries/joomla/twitter/friends.php:235]: Alignment.

File: libraries/joomla/twitter/lists.php [hunk 7]
```diff
@@ -65,22 +65,22 @@ public function getLists($user, $reverse = null)
...
-	 * 								   of metadata about the tweet in a discreet structure, including: user_mentions, urls, and hashtags.
-	 * @param   boolean  $include_rts  When set to either true, t or 1, the list timeline will contain native retweets (if they exist) in addition
-	 * 								   to the standard stream of tweets.
+	 * @param   mixed    $list        Either an integer containing the list ID or a string containing the list slug.
+	 * @param   mixed    $owner       Either an integer containing the user ID or a string containing the screen name.
+	 * @param   integer  $sinceId     Returns results with an ID greater than (that is, more recent than) the specified ID.
+	 * @param   integer  $maxId       Returns results with an ID less than (that is, older than) or equal to the specified ID.
+	 * @param   integer  $count       Specifies the number of results to retrieve per "page."
+	 * @param   boolean  $entities    When set to either true, t or 1, each tweet will include a node called "entities". This node offers a variety
+	 * 								  of metadata about the tweet in a discreet structure, including: user_mentions, urls, and hashtags.
```
Comment (Quy) [inline libraries/joomla/twitter/lists.php:74]: Alignment.

File: libraries/joomla/twitter/lists.php [hunk 8]
```diff
@@ -155,19 +155,19 @@ public function getStatuses($list, $owner = null, $since_id = 0, $max_id = 0, $c
...
-	 * @param   integer  $cursor       Breaks the results into pages. A single page contains 20 lists. Provide a value of -1 to begin paging.
-	 * @param   boolean  $entities     When set to either true, t or 1, each tweet will include a node called "entities". This node offers a variety
-	 * 								   of metadata about the tweet in a discreet structure, including: user_mentions, urls, and hashtags.
-	 * @param   boolean  $skip_status  When set to either true, t or 1 statuses will not be included in the returned user objects.
+	 * @param   mixed    $list        Either an integer containing the list ID or a string containing the list slug.
+	 * @param   mixed    $owner       Either an integer containing the user ID or a string containing the screen name.
+	 * @param   integer  $cursor      Breaks the results into pages. A single page contains 20 lists. Provide a value of -1 to begin paging.
+	 * @param   boolean  $entities    When set to either true, t or 1, each tweet will include a node called "entities". This node offers a variety
+	 * 								  of metadata about the tweet in a discreet structure, including: user_mentions, urls, and hashtags.
```
Comment (Quy) [inline libraries/joomla/twitter/lists.php:162]: Alignment.

File: libraries/joomla/twitter/lists.php [hunk 9]
```diff
@@ -351,19 +351,19 @@ public function subscribe($list, $owner = null)
...
-	 * @param   mixed    $owner        Either an integer containing the user ID or a string containing the screen name of the owner.
-	 * @param   boolean  $entities     When set to either true, t or 1, each tweet will include a node called "entities". This node offers a
-	 * 								   variety of metadata about the tweet in a discreet structure, including: user_mentions, urls, and hashtags.
-	 * @param   boolean  $skip_status  When set to either true, t or 1 statuses will not be included in the returned user objects.
+	 * @param   mixed    $list        Either an integer containing the list ID or a string containing the list slug.
+	 * @param   mixed    $user        Either an integer containing the user ID or a string containing the screen name of the user to remove.
+	 * @param   mixed    $owner       Either an integer containing the user ID or a string containing the screen name of the owner.
+	 * @param   boolean  $entities    When set to either true, t or 1, each tweet will include a node called "entities". This node offers a
+	 * 								  variety of metadata about the tweet in a discreet structure, including: user_mentions, urls, and hashtags.
```
Comment (Quy) [inline libraries/joomla/twitter/lists.php:358]: Alignment.

File: libraries/joomla/twitter/lists.php [hunk 10]
```diff
@@ -434,19 +434,19 @@ public function isMember($list, $user, $owner = null, $entities = null, $skip_st
...
```
Comment (Quy) [inline libraries/joomla/twitter/lists.php:441]: Alignment.

File: libraries/joomla/twitter/statuses.php [hunk 11]
```diff
@@ -563,29 +563,29 @@ public function tweetWithMedia($status, $media, $in_reply_to_status_id = null, $
...
-	 *                                 In cases where a page already includes widgets.js, setting this value to true will prevent a redundant
-	 *                                 script element from being included.
-	 * @param   string   $align        Specifies whether the embedded Tweet should be left aligned, right aligned, or centered in the page.
-	 *                                 Valid values are left, right, center, and none.
-	 * @param   string   $related      A value for the TWT related parameter, as described in Web Intents. This value will be forwarded to all
-	 *                                 Web Intents calls.
-	 * @param   string   $lang         Language code for the rendered embed. This will affect the text and localization of the rendered HTML.
+	 * @param   integer  $id          The Tweet/status ID to return embed code for.
+	 * @param   string   $url         The URL of the Tweet/status to be embedded.
+	 * @param   integer  $maxwidth    The maximum width in pixels that the embed should be rendered at. This value is constrained to be
```
Comment (Quy) [inline libraries/joomla/twitter/statuses.php]: `maxWidth`?

Comment (SharkyKZ) [reply libraries/joomla/twitter/statuses.php]: Renamed.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 7c544d3a0442453b18b044f0e0f7b00b304eb677<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31168">issues.joomla.org/tracker/joomla-cms/31168</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 7c544d3a0442453b18b044f0e0f7b00b304eb677<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31168">issues.joomla.org/tracker/joomla-cms/31168</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31168">issues.joomla.org/tracker/joomla-cms/31168</a>.</sub>

**Pull request merged**